### PR TITLE
increase verbosity of LLVM build script

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by giordano** [comment]
Can we take the opportunity of this PR to add `${prefix}/tools/clang` as an `ExecutableProduct` so that it can be directly called from Julia to compile?

**Comment by vchuravy** [comment]
> Can we take the opportunity of this PR to add `${prefix}/tools/clang` as an `ExecutableProduct` so that it can be directly called from Julia to compile?

Of course! I also added `llvm-mca` for MCAnalyzer.jl

File: L/LLVM/common.jl [hunk 1]
```diff
@@ -243,11 +245,25 @@ function configure(version; assert=false)
...
+    if version >= v"8"
+        push!(products, ExecutableProduct("llvm-mca", :llvm_mca, "tools"))
+    end
+
     config = "LLVM_MAJ_VER=$(version.major)\n"
     if assert
         config *= "ASSERTS=1\n"
     end
-    sources, config * buildscript
+    sources, config * buildscript, products
```
Comment (giordano) [inline L/LLVM/common.jl]: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=991&view=logs&j=96c13d9d-dfd2-51e5-2f76-4a0b3ce09abc&t=09532434-8b0a-5f0e-118d-a3181cde5ec8&l=30
```
ERROR: LoadError: LoadError: syntax: missing separator in array expression
Stacktrace:
 [1] top-level scope at /workspace/L/LLVM/common.jl:254
 [2] include at ./boot.jl:328 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1105
 [4] include(::Module, ::String) at ./Base.jl:31
 [5] include(::String) at ./client.jl:424
 [6] top-level scope at /workspace/L/LLVM/v9/build_tarballs.jl:3
 [7] include at ./boot.jl:328 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1105
 [9] include(::Module, ::String) at ./Base.jl:31
 [10] exec_options(::Base.JLOptions) at ./client.jl:287
 [11] _start() at ./client.jl:460
in expression starting at /workspace/L/LLVM/common.jl:254
in expression starting at /workspace/L/LLVM/v9/build_tarballs.jl:3
```

Comment (giordano) [reply L/LLVM/common.jl]: well, wrong line :sweat_smile:

**Comment by vchuravy** [comment]
Oops

**Pull request merged**